financial-services-accelerator/consent-mgt-api/org.wso2.financial.services.accelerator.consent.mgt.api.service/src/main/java/org/wso2/financial/services/accelerator/consent/mgt/api/service/util/ConsentCoreServiceUtil.java

```diff
@@ -0,0 +1,555 @@
+/**
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ * <p>
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.financial.services.accelerator.consent.mgt.api.service.util;
+
+import com.google.gson.Gson;
+import net.minidev.json.JSONObject;
+import net.minidev.json.parser.JSONParser;
+import net.minidev.json.parser.ParseException;
+import org.apache.commons.collections4.MapUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.financial.services.accelerator.consent.mgt.api.dao.ConsentCoreDAO;
+import org.wso2.financial.services.accelerator.consent.mgt.api.dao.constants.ConsentError;
+import org.wso2.financial.services.accelerator.consent.mgt.api.dao.exceptions.ConsentDataDeletionException;
+import org.wso2.financial.services.accelerator.consent.mgt.api.dao.exceptions.ConsentDataInsertionException;
+import org.wso2.financial.services.accelerator.consent.mgt.api.dao.exceptions.ConsentDataRetrievalException;
+import org.wso2.financial.services.accelerator.consent.mgt.api.dao.exceptions.ConsentMgtException;
+import org.wso2.financial.services.accelerator.consent.mgt.api.dao.models.AuthorizationResource;
+import org.wso2.financial.services.accelerator.consent.mgt.api.dao.models.ConsentAttributes;
+import org.wso2.financial.services.accelerator.consent.mgt.api.dao.models.ConsentHistoryResource;
+import org.wso2.financial.services.accelerator.consent.mgt.api.dao.models.ConsentMappingResource;
+import org.wso2.financial.services.accelerator.consent.mgt.api.dao.models.ConsentResource;
+import org.wso2.financial.services.accelerator.consent.mgt.api.dao.models.ConsentStatusAuditRecord;
+import org.wso2.financial.services.accelerator.consent.mgt.api.dao.models.DetailedConsentResource;
+import org.wso2.financial.services.accelerator.consent.mgt.api.dao.util.DatabaseUtils;
+import org.wso2.financial.services.accelerator.consent.mgt.api.service.constants.ConsentCoreServiceConstants;
+import org.wso2.financial.services.accelerator.consent.mgt.api.service.impl.ConsentCoreServiceImpl;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+/**
```

Comment:
Let's format the classes

```suggestion
import java.util.UUID;

/**
```

Reply:
resolved in https://github.com/wso2/financial-services-accelerator/commit/391597322f04656a0148e55574bbab32386d670d